mk/config.mk

```diff
@@ -180,6 +180,18 @@ CFG_RPMB_FS_RD_ENTRIES ?= 8
 # in case the cache is too small to hold all elements when traversing.
 CFG_RPMB_FS_CACHE_ENTRIES ?= 0
 
+# Print RPMB data frames sent to and received from the RPMB device
+CFG_RPMB_FS_DEBUG_DATA ?= n
+
+# Disable RPMB MAC authentication when CFG_RPMB_FS_NO_MAC=y
+CFG_RPMB_FS_NO_MAC ?= n
```

Comment:
This is negative logic, I would have preferred `CFG_RPMC_FS_MAC ?= y`.

Reply:
True but on the other hand it is desirable to limit the number of CFG_ variables defined during build, and since `=n` is equivalent to unset (both mean feature disabled), there is an advantage in having the most commonly used configurations be disabled.

Reply:
That is low prio compared to making things easier to understand.

Reply:
I'm fine we change this to CFG_RPMB_FS_MAC despite legacy of existing CFG_RPMB_FS_NO_MAC since it's a debug switch.

Reply:
Agree.

Reply:
Sounds good to me.

Reply:
Fine with me.